### PR TITLE
Us99269/avatar

### PR DESCRIPTION
**Pull request by mtjandra**
* Ability to display an avatar for the profile image
* If failing to load profile image fallback to initials

### CODE REVIEW

**Comment by svanherk** [comment]
I'm seeing the same issue with `imageUrl` as in Issue https://github.com/BrightspaceUI/profile-image/issues/10, updating the url doesn't load the new image in Edge or Firefox (did not test Safari)

**Comment by mtjandra** [comment]
> I'm seeing the same issue with imageUrl as in Issue #10, updating the url doesn't load the new image in Edge or Firefox (did not test Safari)

Yeah I'll be investigating. Even the observer doesn't trigger on change so not too sure what is happening

**Comment by mtjandra** [comment]
@svanherk I think it is a browser problem not one with polymer. If you change the image or initials with javascript it updates no problem. Ex:
```
var y = document.querySelector('#y');
y.imageUrl = 'invalid'; //Image Updates

var x = document.querySelector('#x');
x.firstName = "OK" // Initials Update
```

I did the above in the console

File: d2l-profile-image.html [hunk 1]
```diff
@@ -36,18 +40,22 @@
...
 		</template>
 
-		<template is="dom-if" if="[[_showInitials(firstName, colourId)]]">
+		<template is="dom-if" if="[[_showInitials(firstName, colourId, imageUrl, _failedToLoadImage)]]">
 			<div id="initials" class="initials">
 				[[_getInitials(firstName, lastName)]]
 			</div>
 		</template>
 
+		<template is="dom-if" if="[[_showAvatar(imageUrl, _failedToLoadImage)]]">
```
Comment (svanherk) [inline d2l-profile-image.html]: The cascading show logic of the different `dom-if`s feels like its getting a bit messy, but I think there's a follow-up PR coming to adjust this.

Comment (mtjandra) [reply d2l-profile-image.html]: Yup there is I'll post that one next

**Pull request merged**